tools/scripts/update-native-event-names.ts

```diff
@@ -0,0 +1,88 @@
+import { writeFileSync } from 'node:fs';
+import path from 'node:path';
+import chalk from 'chalk';
+import bcd from '@mdn/browser-compat-data';
+
+run().catch((ex) => {
+  console.error(chalk.red(ex));
+  process.exit(1);
+});
+
+async function run(): Promise<void> {
+  const names = getEventNamesFromBrowserCompatData();
+  const fileName = path.resolve(
+    __dirname,
+    '../../packages/utils/src/eslint-plugin/get-native-event-names.ts',
+  );
+
+  process.stdout.write(chalk.cyan(`Updating ${path.basename(fileName)}...\n`));
+
+  writeFileSync(
+    fileName,
+    `
+let nativeEventNames: ReadonlySet<string> | null = null;
+
+/**
```

Comment:
This is looking really good now, thanks a lot!

Final request would be to make it clear that this file should not be maintained by hand and what command should be run instead.

E.g. exactly like https://github.com/angular-eslint/angular-eslint/blob/53a2afe28e0d7dce23d6a3f974abaad76b1b1c66/packages/angular-eslint/src/configs/template-accessibility.ts#L1-L5 and other generated configs